bot/cogs/help_channels.py

```diff
@@ -680,12 +684,12 @@ async def on_message(self, message: discord.Message) -> None:
             await self.move_to_in_use(channel)
             await self.revoke_send_permissions(message.author)
             # Add user with channel for dormant check.
-            self.help_channel_claimants[channel] = message.author
+            await self.help_channel_claimants.set(channel.id, message.author.id)
 
             self.bot.stats.incr("help.claimed")
 
-            self.claim_times[channel.id] = datetime.now()
-            self.unanswered[channel.id] = True
+            await self.claim_times.set(channel.id, str(datetime.now()))
```

Comment:
Something irks me about storing timestamps as strings. Using a POSIX timestamps (i.e. relative to the epoch) seems more reliable. By the way, should this be in UTC or does it not really matter as long as it's also decoded the same way?

Reply:
> Using a POSIX timestamps (i.e. relative to the epoch) seems more reliable
I don't really mind either approach, but I'm fine with changing it to do this.

> By the way, should this be in UTC
I can't see that it should matter. Can you think of an edge case where it would?

Reply:
I think we use UTC based time everywhere in the bot, it would be nice for consistency and in case if debugging.

Reply:
That's not true; we both use `datetime.datetime.now` and `datetime.datetime.utcnow` and even in this very extension, `datetime.datetime.now` is used. Both return naive datetime objects without tz information and they both currently return the same time. Granted, the latter could change if were to change the local time settings of the VPS.

Reply:
I could change these all to `utcnow` since they are more consistent.

Reply:
Interesting how different opinions we have on this. My approach would have been `isoformat` and `fromisoformat` using UTC, the string representation requires importing `dateutil` which is fine of course, but it still feels like an extra step (and I personally don't know what `str(dt_instance)` looks like at all).